sim-lib/src/lib.rs

```diff
@@ -535,7 +535,7 @@ async fn write_payment_results(
                 match payment_report {
                     Some((details, result)) => {
                         result_logger.report_result(&details, &result);
-                        log::trace!("Resolved payment received: ({:?}, {:?})", details, result);
+                        log::trace!("Resolved payment received: ({:?}, {:?}).", details, result);
```

Comment:
Ah, nice catch. Updating with `Display` for both struct: 
```
2023-09-14T14:39:23.130Z TRACE [sim_lib] Resolved dispatched payment: Payment 7dfaa107c953877a57cfa8c0478c85abb23853edbb15ef8c03934ba82ab77202 dispatched at 2023-09-14 14:39:23 sending 100000 msat from 0360c03678098b958532fa2a0b969969d2f8e87a33b8c6116fcf8a54f8eda9c3e9 -> 030c3d33944e8fc7435f7bd45414910601349398ef1d5543074c4f1cac603f9628 with: Payment outcome: RouteNotFound with 0 htlcs.

```

Reply:
```
dispatched at 2023-09-14 14:39:23
```

This makes me wonder: do we want to log the real time, or the time relative to the start of the simulation?

Reply:
I think _for now_ we want real time because that makes the payment easy to cross-reference with lookup on the actual lightning node? If we do simulation time we should probably do it relative to the "time system" that's being used?

Although one Q I did have here is whether we want to do a pretty date or just a unix timestamp? Because (I think) most things are surfaced with unix ts on the actual lightning node.

Reply:
I think given this is `trace`, unix ts should be fine. This may also save a dependency (`chrono`), which is always nice.

Reply:
Yeah I agree, switched to unix in a fixup https://github.com/bitcoin-dev-project/sim-ln/pull/86/commits/9bf1e5369827bc428a9e9801ae1caf122469476f